File: backend/models/errorlog.go
package models

import (
	"github.com/jinzhu/gorm"
)

type ErrorLog struct {
	gorm.Model

	message        string
	stacktrace     string
	innerException string
	statusCode     int32
	typeException  string
	DeviceID       int
	Type           string
	dateTime       int32
}

File: core_backend/core_backend/Models/ErrorLog.cs
namespace core_backend.Models
{
    public class ErrorLog : BaseModel
    {
        public ErrorLog() { }

        public string StackTrace { get; set; }
        public string Message { get; set; }
        public string InnerException { get; set; }
        public string Body { get; set; }
        public int StatusCode { get; set; }
        public DateTime DateTime { get; set; }
        public string TypeException { get; set; }
        public string UserId { get; set; }
    }
}
